.gitignore

```diff
@@ -33,3 +33,4 @@ go.sum
 /anax-in-container/LICENSE.txt
 /agent-install/test/config/agent-install.cfg                                                                                                                               /agent-install/test/config/agent-install.crt                                                                                                                               /agent-install/test/config/switch/agent-install.cfg                                                                                                                        /agent-install/test/config/switch/agent-install.crt
 /agent-install/test/config/switch/agent-install-test-results.log
+cli/horizon/
```

Comment:
why is this added?

Reply:
sorry this is a mistake, please ignore this